src/main/ts/components/Editor.tsx

```diff
@@ -46,9 +46,10 @@ export class Editor extends React.Component<IAllProps> {
     cloudChannel: '5'
   };
 
+  public editor?: TinyMCEEditor;
```

Comment:
Why is this made public?

Reply:
Mainly because it's the easiest way to give people a reference to TinyMCE. As in you can do this:
```tsx
const MyEditor = () => {
  const tinyRef = useRef(null);
  const getContent = () => tiny.current && console.log(tinyRef.current.editor.getContent());
  return (
  <>
    <button onclick={getContent}>Print content to console</button>
    <Editor ref={tinyRef} />
  </>
  );
}
```
Also because if you use the integration from Javascript you can do that anyway even when it is marked `private` as the compiler does not make any effort to hide the value so the `private` was only making it more annoying for our typescript users.